### ISSUE TITLE
firefox, bug with sync and multi profile

### ISSUE THREAD

**Issue by Grummfy** [wallabag/wallabagger#129]
Hello,

## Context :
I have my firefox profile sync with 3 firefox instances. I configure wallabager on a computer, and it was working perfectly!
So now, I switch from computer and try to save an article. I got no error alert, but it's like if the extension try to save but without succeeding to do it. Then I realize that I haven't made any configuration to this firefox instance. ;(
I import the configuration file from the preferences, then it has working perfectly directly ;)

## Possible solution:
* provide an alert for this specific case (I suppose that when it's not configured, there is an alert ?)
* use the password manager of firefox to save the tokens and profil to let it sync to other computers

thanks

**Comment by Rurik19** [comment]
I wouldn't go so far as to call ~~the brother fat~~ this a bug :smile: We didn't declare any syncing functionality. But it is an interesting feature request

**Comment by Grummfy** [comment]
the issue here is that there is no alert if it's logged, I didn't try on a fresh install of wallabager ... but perhaps it's the same

**Comment by Rurik19** [comment]
Maybe I don't understand well:  what does "I switch from computer" mean?

**Comment by Grummfy** [comment]
There is several things in my issue.

1) the fact that when you have a firefox profile sync between several computer, the firefox instance (on other computer) doesn't put any warning when we try to sae a new page
2) perhaps this is the same behavior for a new installation (I din't test it)
3) A new feature to make the extension, with sync profile working, would be to sync the data

Is it clearer?

**Comment by Rurik19** [comment]
Yes. Of course, the other computer has a new installation. The message about not completed setup would be another useful feature.

**Comment by prcmt** [comment]
Chrome, Firefox, and Edge support the `sync` storage area that is synced between devices.

In theory, taking advantage of this would be a simple case of changing the two instances of `storage.local` to `storage.sync` in [wallabag-api.js](https://github.com/wallabag/wallabagger/blob/786d4265d0287ae4e07fbf1e3db85def79239280/wallabagger/js/wallabag-api.js#L41-L46).

**Comment by AlejandroAkbal** [comment]
Any update? Would be a great UX improvement, specially now that people are migrating from getPocket
